src/components/SingleReview.jsx

```diff
@@ -0,0 +1,33 @@
+import { useParams } from "react-router-dom";
+import { getReviewsById } from "../requests";
+import { useEffect } from "react";
+
+const SingleReview = ({ review, setReview }) => {
+  let review_id = useParams().review_id;
+  useEffect(() => {
+    getReviewsById(review_id).then((data) => {
+      setReview(data);
+    });
+  }, []);
+
+  return (
+    <section>
+      <p>{review.owner}</p>
```

Comment:
You may want to add some labels here to indicate what some of these mean for a user